stack-nightly.yaml

```diff
@@ -0,0 +1,11 @@
+resolver: nightly-2022-03-25
```

Comment:
So does this get overwritten by `--resolver nightly`?

Reply:
Yeah, unfortunately `resolver: nightly` in a `stack.yaml` doesn't work. So this basically serves as "when was the last time we worked on this?" and you must use `--resolver` to test actual now-nightly.